docs/reference/settings/security-settings.asciidoc

```diff
@@ -445,12 +445,20 @@ An `s` at the end indicates seconds, or `ms` indicates milliseconds.
 Defaults to `5s` (5 seconds ).
 
 `timeout.tcp_read`::
-The TCP read timeout period after establishing an LDAP connection.
-An `s` at the end indicates seconds, or `ms` indicates milliseconds.
-Defaults to `5s` (5 seconds ).
+deprecated[7.5] The TCP read timeout period after establishing an LDAP
+connection. This is equivalent to and is deprecated in favor of
+`timeout.response` and they cannot be used simultaneously. An `s` at the end
+indicates seconds, or `ms` indicates milliseconds. Defaults to the value of
+`timeout.ldap_search`.
```

Comment:
I'd propose that we remove the "Defaults to ..." from here, since it is the deprecated setting, it doesn't really have a default.

Reply:
Good point! If you one doesn't specify the deprecated settings, it's the new settings that takes a default value, the old one is just deprecated.